src/components/Clock.jsx

```diff
@@ -39,10 +44,32 @@ var Clock = React.createClass({
             transform: 'rotate(' + secondsScale(this.state.seconds) + 'deg)'
         };
 
+        // Day/night indicator
+        var isDay = true;
+        if (this.state.moment.hours() >= 18 || this.state.moment.hours() < 6) {
```

Comment:
You mean the the values 6 and 18 (approximate start and end of the daylight)? Yes, those values changes a lot: I live in up here in Finland where the amount of daylight changes constantly. So, fixed values can surely be parametrized but I don't think one keeps adjusting them throughout the year :) Fetching the _actual_ info from somewhere would be nice thouch, though.

Anyway, I'll add them as params if that's what you're after.